src/pages/K8s.vue

```diff
@@ -363,7 +363,7 @@ export default {
     async handleDisableKubernetesCheckbox(value) {
       if (value !== this.settings.kubernetes.enabled) {
         const confirmationMessage = [
-          `${ value ? 'Enabling' : 'Disabling' } Kubernetes requires a restart. `,
+          `${ value ? 'Enabling' : 'Disabling' } Kubernetes requires a VM restart. `,
```

Comment:
A bit of a nitpick, but I think something like this sounds a little better. What do you think?

```suggestion
          `${ value ? 'Enabling' : 'Disabling' } Kubernetes will restart the virtual machine. `,
```

Reply:
@rak-phillip maybe `will require restarting the virtual machine` ?
Since is an action that the user has to Accept. What do you think?

Reply:
Looks good 🚀